src/components/searchBar/SearchBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import  { ReactComponent as SearchIcon } from "../../assets/iconSearch.svg";
+
+export const SearchBar = ({ onChangeHandler }) => {
+    const [visibleInput, setVisibleInput] =useState(false);
+
+    const searchInputHandler = () =>{
+        visibleInput ? setVisibleInput(false) : setVisibleInput(true);
+    };
+
+  return (
+      <>
+    <div className="searchbar-container">
+        <SearchIcon onClick={searchInputHandler}  className="searchbar-container--icon"/>
```

Comment:
Are you sure you are utilising BEM methodology right? I think you should use modifier names `--modifier` when you change the appearance or the behaviour of blocks and elements. WDYT?

Reply:
I can see your point and you are right because typically modifier classes are added to blocks/elements they modify, and keep the original class. 
But as far as I have seen and according to the seminars I have participated this practice can be used effectively.